Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -13,7 +13,9 @@ public enum Route: DerivePartialIsos {
   case admin(Admin)
   case cancel
   case confirmCancel
+  case confirmEmailChange(userId: Database.User.Id, emailAddress: EmailAddress)
   case episode(Either<String, Int>)
+  case expressUnsubscribe(userId: Database.User.Id, newsletter: Database.EmailSetting.Newsletter)
```

Comment:
new endpoint for unsubscribing from a newsletter directly, i.e. not requiring to be logged in